internal/fwserver/server_importresourcestate.go

```diff
@@ -35,13 +35,16 @@ type ImportResourceStateRequest struct {
 	// TypeName is the resource type name, which is necessary for populating
 	// the ImportedResource TypeName of the ImportResourceStateResponse.
 	TypeName string
+
+	ClientCapabilities *resource.ImportStateClientCapabilities
```

Comment:
Similar to feedback above, if we do the defaulting earlier in `fromproto`, this can just be:
```go
ClientCapabilities resource.ImportStateClientCapabilities
```